### PR TITLE
added a little extra documentation

### PR DESCRIPTION
**Pull request by titaniumbones**
just makes it a little easier for the ignorant like me!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,25 @@
 Here's a little repo to allow you to fork all U.S. government repos, just in case the originals disappear in the next few years.
-If you have a git username and ouath key, you can fork the repos yourself.
+
+If you have a git username and ouath key, you can fork the repos yourself. Running this script will create ~1670 new repositoriies in your account, so you may want to create a special account just for this. You will need a [new token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/) for the new account. Be sure to copy the new key when you create it, as Github will not show it to you again.
```
Comment (jeveleth) [inline README.md]: "repositoriies" should be "repositories"

File: README.md [hunk 2]
```diff
@@ -1,18 +1,25 @@
...
 ```
 
 To update your forked repos:
 ```
+export GH_USERNAME='Your Username here'
+export GH_OAUTH_KEY='Your OAUTH Key Here'
 php run.php update
 ```
 
+(Obviously you can use also  some other mechanism to set the relevnat environment variables.)
```
Comment (jeveleth) [inline README.md]: "relevnat" should be "relevant"

**Review by jeveleth — CHANGES_REQUESTED** [review]
Looks good. Added a couple of comments.

**Review by jeveleth — APPROVED** [review]

**Comment by titaniumbones** [comment]
The PR is still open but I think all changes have been approved. I haven't used change requests much so tell me if I'm doing something wrong...

**Comment by jeveleth** [comment]
Looks good. Thanks for the documentation!

**Pull request merged**